### ISSUE TITLE
[Suggestion] Config for wraiths hovering above fluids

### ISSUE THREAD

**Issue by AquaMagic** [elucent/eidolon#77]
Wraiths being unaffected by fluids means they cannot be flushed/moved like other mobs in mob grinders. This means that, at best, they hang around and fill up the mob cap and at worst clog up mob farms which rely on water to sort mobs, breaking certain parts of them by body-blocking other mobs. It would be good to have a config option to allow wraiths to be pushed by fluids like witches or creepers, or to have them sink in water like other undead. Alternatively, optionally making the spawn conditions for wraiths more specific (e.g. making them only spawn in cold biomes, making them taller than zombies) would also solve this problem, so one could make farms where they don't spawn at all.

**Comment by elucent** [comment]
Build a better mob farm.

**Comment by AquaMagic** [comment]
It's not that wraiths are impossible to deal with, but they add a layer of complexity to mob farm design that I'd rather be able to turn off.

I ran into this issue because I was trying to make a zombie->drowned converting mob farm, which is simply done with a water chute. Creepers and witches would float on the water chute, so I added a stream on top of the chute to push them aside, while zombies and skeletons fall through. Wraiths just sit on top of the stream, filling up the mob cap and deflecting zombies into the wrong chute. I know I _could_ "build a better mob farm." I could add another single block of water above the chute, so faster-falling mobs fall through to the water chute, while wraiths (and chicken jockeys) are caught by it. Then I need to add a mechanism to kill them so they don't fill the mob cap, while also avoiding disrupting the flow of other mobs. Maybe wraiths are attracted by turtle eggs, villagers or iron golems and I could use those instead.

These are not the only solutions, but that's not the point. The point is, the hovering mechanic needlessly complicates otherwise fairly straightforward mob farm designs (not just this one). Again, it would be good to have a config option to turn that mechanic off.
